tutorials.html

```diff
@@ -49,20 +49,28 @@ <h3>CDMS</h3>
     <div class="container">
         <h3>VCS</h3>
         <p> Examples of how to use VCS</p>
+        <div class="example">
+            <a href="Jupyter-notebooks/vcs/VCS_Basics/VCS_Basics.html">
+                <div class="img-wrapper">
+                    <img src="VCS_Basics.png"/>
```

Comment:
```suggestion
                    <img src="Jupyter-notebooks/vcs/VCS_Basics/VCS_Basics.png"/>
```

Reply:
Thanks @Mauzey1. With the new .pngs on a different branch that had not been deployed when I first tested the page, it was more difficult for me to catch this mistake. I believe it should be fixed with the second commit for this PR.